ADVANCED.md

```diff
@@ -275,17 +275,17 @@ import { SearchBox } from "@elastic/react-search-ui";
+| view                          | Component                                                                    | no        | [SearchBox](packages/react-search-ui-views/src/SearchBox.js)       |         | Used to override the default view for this Component. See the [Customization: Component views and HTML](#component-views-and-html) section for more information.                                                                                                                            |
+| autocompleteResults           | Boolean or [AutocompleteResultsOptions](#AutocompleteResultsOptions)         | Object    | no                                                                 |         | Configure and autocomplete search results. Boolean option is primarily available for implementing custom views.                                                                                                                                                                             |  |
+| autocompleteQuerySuggestions  | Boolean or [AutocompleteSuggestionsOptions](#AutocompleteSuggestionsOptions) | Object    | no                                                                 |         | Configure and autocomplete query suggestions. Boolean option is primarily available for implementing custom views.                                                                                                                                                                          |
+| autocompleteMinimumCharacters | Integer                                                                      | no        | 0                                                                  |         | Minimum number of characters before autocompleting.                                                                                                                                                                                                                                         |
+| autocompleteView              | Render Function                                                              | no        | [Autocomplete](packages/react-search-ui-views/src/Autocomplete.js) |         | Provide a different view just for the autocomplete dropdown.                                                                                                                                                                                                                                |
+| onSelectAutocomplete          | Function(selection. options, defaultOnSelectAutocomplete)                    | no        |                                                                    |         | Allows overriding behavior when selected, to avoid creating an entirely new view. In addition ot the current `selection`, various helpers are passed as `options` are passed as the second parameter, and the `onSelectAutocomplete` in order to simply wrap and call the default behavior. |
```

Comment:
> In addition ot the current `selection`, various helpers are passed as `options` are passed as the second parameter, and the `onSelectAutocomplete` in order to simply wrap and call the default behavior.

Perhaps: 

> In addition to the current `selection`, various helpers are passed as `options` to the second parameter - and the `onSelectAutoComplete` - to wrap and call the default behaviour.

Reply:
I butchered that wording. How about this?

> In addition to the current `selection`, various helpers are passed as `options` to the second parameter. This third parameter is the default `onSelectAutocomplete`, which allows you to defer to the original behavior.

Reply:
❤️